### ISSUE TITLE
Beadandó

### ISSUE THREAD

**Issue by nemetha17** [Uni-Sopron/fonya-o21-beadndo-regex-comparison#1]
Elkészült a beadandóm HTML-ben. Rá néznél, hogy mit kellene javítani benne, ezek a reguláris kifejezések megfelelnek-e vagy összetettebbek kellenének?

**Comment by nemetha17** [comment]
@hegyhati Rá tudnál nézni, hogy mit kellenék javítani benne?

**Comment by hegyhati** [comment]
Amint eljutok ide, ra fogok, de lehet csak holnap, holnaputan.

**Comment by nemetha17** [comment]
@hegyhati Bocsánat, hogy megint megjelöllek. Esetleg sikerült rá nézned, mivel úgy tudom 17.-ig kellene nekem megkapnom a jegyet, hogy tudjak majd záróvizsgázni.

**Comment by hegyhati** [comment]
Igen, tudom, prio lista elejen vagy, csak sajnos tegnap el kellett utaznom Sopronbol, es ebben a pillanatban ertem vissza. Most kifujom magam kicsit, meg par dolgot elrendezek, de utana  a Te beadandod jon. Ma este meg jegy lesz ra adva.

**Comment by hegyhati** [comment]
No, mivel most ennyire last minute vagyunk, elfogadom, es beirom az otost, de... tegy meg majd nekem annyit, ohogy ezt a doksit megcsinalod valamikor normalisra. Kodreszletet kepkent bevagni.... finoman szolva favago. Marcsak itt, szimpla md-ben is szebbet/atlathatobbat lehet csinalni:

```python
count = 0
for line in open("log.txt"):
  if re.search("^2016-09-09", line):
    x += 1
```

A js-es dolgot nem teljesen ertem, miert `\b`-t kerestettel es nem `$`-et? Ha jol ertem, sortorest keresel, ami kulonbozoen fordulhat elo kulonbozoen keszitett fajlokban.
